apps/explorer/src/main.tsx

```diff
@@ -1,24 +1,10 @@
-import * as Sentry from '@sentry/react';
-import { BrowserTracing } from '@sentry/tracing';
 import { createRoot } from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import './styles.css';
 
 import App from './app/app';
-import { ENV } from './app/config/env';
 import { StrictMode } from 'react';
 
-const { dsn } = ENV;
-
-/* istanbul ignore next */
-if (dsn) {
```

Comment:
question (blocking): Why remove this and put it in the useEffect. I don't love it being in an effect

Reply:
because we don't know the env here, since it comes from a hook